src/org/ods/component/Pipeline.groovy

```diff
@@ -211,6 +211,7 @@ class Pipeline implements Serializable {
                 containers: config.podContainers,
                 volumes: config.podVolumes,
                 serviceAccount: config.podServiceAccount,
+                annotations: config.annotations,
```

Comment:
I think there is a similar place for the orchestration pipeline

Reply:
Thanks for the hint!!
We also have it in Quickstarter pipeline